udata_front/theme/gouvfr/assets/js/components/ReadMore/ReadMore.vue

```diff
@@ -67,36 +61,37 @@ export default defineComponent({
 
     const containerHeight = ref(DEFAULT_HEIGHT);
 
-    const updateReadMoreHeight = () => {
-      let contentHeight = Array.from(container.value?.children || [])
```

Comment:
Some changes here in the logic.
I think I had some issues expanding: it did not expand to the entire description or message.
May be related, but I'm not sure that toggles introduced in markdown are being treated correctly.

Reply:
Yep, I forgot to change the ref on some part of the logic